README.md

```diff
@@ -1 +1,314 @@
-# signal-searcher
\ No newline at end of file
+# Requirements
+
+**Signal Searcher attempts to find interesting and important signals in MLab's Internet
+measurement data using computers instead of people.**
+
+[Measurement Lab](mlab) is collecting lots and lots of measurement data from all over the Internet. The project is drowning in data, and too frequently has to work too hard to acquire any of data's derivatives: information, knowledge, and wisdom.  In the past, issues were found because there were operator rumors of bad behavior. Depending on intensive investigations by humans spurred by rumor-mill dross is not a sustainable solution. As a process it is too random to get started reliably, and for its execution it depends on a very scarce resource (researcher time).
```

Comment:
Is this correct that we are drowning in data? Is the data sufficient enough to support claims outside of the United States, see: Turkey during the couple. Who is the researcher in this case?